frontend/src/components/TournamentTeam.js

```diff
@@ -36,6 +37,22 @@ const TournamentTeam = () => {
   const currentTeamNo = match =>
     match.team_1.ultimate_central_slug === params.team_slug ? 1 : 2;
 
+  createEffect(() => {
+    if (matchesQuery.status === "success" && !matchesQuery.data?.message) {
+      const teamMatches = matchesQuery.data?.filter(
+        match =>
+          match.team_1?.ultimate_central_slug === params.team_slug ||
+          match.team_2?.ultimate_central_slug === params.team_slug
+      );
+
+      setMatchesGroupedByDate(
+        Object.groupBy(teamMatches, ({ time }) =>
+          new Date(Date.parse(time)).toLocaleDateString()
```

Comment:
If the user is in a different timezone, the matches on the same "day" can get grouped as different days? :thinking:

Reply:
Would Date.prototype.getUTCDate() work ?

Reply:
`getUTCDate` only gets the day, right?

I think you probably just need to use these?

```js
const year = date.getUTCFullYear(); 
const month = date.getUTCMonth() + 1; 
const day = date.getUTCDate(); 
```

Reply:
Oh, well, since you are only grouping, `getUTCDate` should work fine. Unless a tournament lasts for more than a month, where the dates clash.

Reply:
Wow I sent my message before yours but mine updated later. Super internet I have 👀

Reply:
I'm outside now, I'll update and send in some time